packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -205,7 +205,7 @@ export default async (cmd: Command) => {
           return chalk.red('Please enter an ID for the plugin');
         } else if (!/^[a-z0-9]+(-[a-z0-9]+)*$/.test(value)) {
           return chalk.red(
-            'Plugin IDs must be kebab-cased and contain only letters, digits, and dashes.',
```

Comment:
Ran into this while testing, since dashes are mentioned later in the same message it seemed more understandable just to say lowercase here.